.github/workflows/update-core-deps.yml

```diff
@@ -31,6 +27,8 @@ jobs:
       # This prevents annoying change when contributors run `npm i` on their local machine.
       # Example: https://github.com/microsoft/TypeScript-DOM-lib-generator/pull/1463
       - run: npm i
+      - name: Update MDN Data
+        run: node scripts/generateMDN
```

Comment:
We are not really generating but more fetching.

really not a fan of camelCasing on file paths though 🫠

I do recognize we have deploy/ which is mostly written by Orta than me, and we do have jsonc files that preexisted, but any new files have been hyphen-cased...